include/dao/dao.js

```diff
@@ -322,8 +322,8 @@ module.exports = function DAOModule(pb) {
                 .skip(offset);
 
             //apply sort order
-            if (options.orderBy) {
-                cursor.sort(options.orderBy);
+            if (options.order) {
+                cursor.sort(options.order);
```

Comment:
The change is in _doQuery(). The underscore indicates that it is intended to be a private function only used inside dao.js, correct? I would think any existing deployments would be calling the q() function in dao.js which was setting the "order" property, not "orderBy".